website/pages/api-docs/operator/scheduler.mdx

```diff
@@ -0,0 +1,157 @@
+| ------ | -------------------------------------- | ------------------ |
+| `GET`  | `/v1/operator/scheduler/configuration` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required    |
+| ---------------- | --------------- |
+| `NO`             | `operator:read` |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    https://localhost:4646/v1/operator/scheduler/configuration
+```
+
+### Sample Response
+
+```json
+{
+  "Index": 5,
+  "KnownLeader": true,
+  "LastContact": 0,
+  "SchedulerConfig": {
+    "CreateIndex": 5,
+    "ModifyIndex": 5,
+    "SchedulerAlgorithm": "spread",
+    "PreemptionConfig": {
+      "SystemSchedulerEnabled": true,
+      "BatchSchedulerEnabled": false,
+      "ServiceSchedulerEnabled": false
+    }
+  }
+}
+```
+
+#### Field Reference
+
+- `Index` `(int)` - The `Index` value is the Raft commit index corresponding to this
+  configuration.
+
+- `SchedulerConfig` `(SchedulerConfig)` - The returned `SchedulerConfig` object has configuration
+  settings mentioned below.
+
+  - `SchedulerAlgorithm` `(string: "binpack")` - Specifies whether scheduler binpacks or spreads allocations on available nodes.
+
+  - `PreemptionConfig` `(PreemptionConfig)` - Options to enable preemption for various schedulers.
+
+    - `SystemSchedulerEnabled` `(bool: true)` - Specifies whether preemption for system jobs is enabled. Note that
+      this defaults to true.
+
+    - `BatchSchedulerEnabled` `(bool: false)` - Specifies whether preemption for batch jobs is enabled. Note that
+      this defaults to false and must be explicitly enabled.
+
+    - `ServiceSchedulerEnabled` `(bool: false)` - Specifies whether preemption for service jobs is enabled. Note that
+      this defaults to false and must be explicitly enabled.
+
+  - `CreateIndex` - The Raft index at which the config was created.
+  - `ModifyIndex` - The Raft index at which the config was modified.
+
+## Update Scheduler Configuration
+
+This endpoint updates the scheduler configuration of the cluster.
+
+| Method        | Path                                   | Produces           |
+| ------------- | -------------------------------------- | ------------------ |
+| `PUT`, `POST` | `/v1/operator/scheduler/configuration` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required     |
+| ---------------- | ---------------- |
+| `NO`             | `operator:write` |
+
+### Bootstrap Configuration Element
+
+The [`default_scheduler_config`][] attribute of the server stanza will provide a
+starting value for this configuration. Once bootstrapped, the value in the
+server state is authoritative.
+
+### Parameters
+
+- `cas` `(int: 0)` - Specifies to use a Check-And-Set operation. The update will
```

Comment:
```suggestion
- `cas` `(int: 0)` - Specifies to use a check-and-set operation. The update will
```

Reply:
The docs seem to reference Check-And-Set elsewhere so hesitant to change, all of these docs changes where cut/paste